sdk/include/stdlib.h

```diff
@@ -4,10 +4,37 @@
 #pragma once
 
 #include <cdefs.h>
+#include <compartment-macros.h>
 #include <stddef.h>
 #include <stdint.h>
 #include <timeout.h>
 
+#ifndef MALLOC_DEFAULT_QUOTA
+#	define MALLOC_DEFAULT_QUOTA 4096
+#endif
+struct MallocCapability
```

Comment:
It's a bit of a shame that the layout of this `struct` has to be known like this, but doing anything else and supporting static construction is a rabbit hole I don't think we want to fall into.  Though if you come up with some brilliance to avoid it...